recipes/http-request/all/conanfile.py

```diff
@@ -0,0 +1,76 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import copy, get
+from conan.tools.layout import basic_layout
+import os
+
+required_conan_version = ">=1.50.0"
+
+
+class CpphttplibConan(ConanFile):
+    name = "http-request"
+    description = "A modern C++ lightweight cross-platform HTTP request library."
+    license = "MIT"
+    homepage = "https://github.com/Nevermore1994/http-request"
+    url = "https://github.com/conan-io/conan-center-index"
+    topics = ("http", "https", "modern C++")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "with_openssl": [True, False]
```

Comment:
In upstream, this option is called `DISABLE_HTTPS`, we might want to have symmetry there :)